_DeploymentAndDistroPackaging/ansible/README.md

```diff
@@ -70,13 +59,27 @@ before proceeding to the next step.
 
 ---
 
+## Start the deployment container with your setup
+```
+$ docker run --privileged -v /path/to/your/.ssh/key:/root/.ssh/id_rsa \
+             -v $(pwd):/root/ciao \
+             -it clearlinux/ciao-deploy
+```
+
 ### Run the playbook
 
     ansible-playbook -i hosts ciao.yml
```

Comment:
If `_DeploymentAndDistroPackaging/ansible/hosts` and `_DeploymentAndDistroPackaging/ansible/group_vars/all` files were edited, I guess it makes sense user would probably be in the `_DeploymentAndDistroPackaging/ansible` directory, however any assumption of $PWD may be wrong, so I'd suggest to instruct the user to map depending on where the user prompt is, for example:

```
## Start the deployment container with your setup
once you have edited the `hosts` file, you must map the
`_DeploymentAndDistroPackaging/ansible` to the container,
so if your current working directory is `ciao`, you would
start the container as follows:

$ docker run --privileged -v /path/to/your/.ssh/key:/root/.ssh/id_rsa \
             -v $(pwd)/_DeploymentAndDistroPackaging/ansible:/root/ciao \
             -it clearlinux/ciao-deploy
```